packages/cli/demo/cat.js

```diff
@@ -874,36 +755,59 @@ const render = value => {
       return $value;
     }
     case 'copyArray': {
-      const $value = document.createElement('div');
-      $value.className = 'array';
+      const $value = document.createElement('span');
+      $value.appendChild(document.createTextNode('['));
+      const $entries = document.createElement('span');
+      $entries.className = 'entries';
+      $value.appendChild($entries);
+      let $entry;
       for (const child of value) {
+        $entry = document.createElement('span');
+        // $entry.className = 'entry';
```

Comment:
There appears to be some leftover commented-out lines in the `copyArray` and `copyRecord` case blocks.